### DISCUSSION TITLE
Camel bottom airlock...

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14005] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The story begins on a saturday, as the fresh vernal breeze entered my room through the window, contrasting with the warmth of my computer room. After perousing my virtual atheneum, I decided to spend a bit of the afternoon experiencing an pleasant journey of fantasy and exoticism through the entertainement software known as "Barotrauma". 

As I was inspecting potental submersible vessel that I would later commandeer through the icy depths of the jovian moon, a detail had caught the corner of my scrupulous eye:

To my dismay I could not help but notice out of the ordinary: by candidness,  the shipwright responsible for assembling this mighty barge had placed a structure colloquially referred to as _"large horizontal inside wall"_. Said hull was firmly wedged between two distinct objects: On its left, the rounded end of the steely carapace evelopping the ship, on its right a sliding hydraulic door, serving as an exit point for any subaquatic explorer brave enough to venture out in the dark unknown.

_"This cannot be right!"_, I exclaimed in stupor, holding back a groan of discontent. Such a rudimentary mistake could not be left unseen: I had to refer to higher powers, the digital demiurges responsible of such a folly. 

"Hear me, Ô gods!", I beseeched, as my eyes sored, my throat clenched and my fists shook. 
"I have but one humble request:"

# Fix this

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/34100c4e-c01e-441d-9a92-cd584efa1d7e)

### Reproduction steps

Camel

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Does this actually cause some kind of problems with the docking or something else, or is it just a visual issue?

**Reply by ubertpendragon** [reply]
Besides the exposed components it's just visual. The people on discord had a few horrified reactions

**Comment by Regalis11** [comment]
Fixed in our private repo, fix coming up in the next unstable build.
